.github/labels.yml

```diff
@@ -0,0 +1,129 @@
+- name: "0 - Backlog"
+  color: "CCCCCC"
+  description: "Issue is accepted, but is not ready to be worked on or not in current sprint"
+- name: "0 - _Triaging"
+  color: "EEEEEE"
+  description: "Issue is accepted, but a milestone has yet to be added for the issue"
+  aliases: ["triage", "_triage", "0 - Triaging"]
+- name: "0 - Waiting on User"
+  color: "AABBDD"
+  description: "Insufficient information for issue or PR, issue may be closed if no response from user"
+- name: "1 - Ready"
+  color: "CCCCCC"
+  description: "Issue is accepted, milestone added and are ready to be worked on"
+- name: "2 - Working"
+  color: "CCCCCC"
+  description: "A user or team member has started working on the issue"
+- name: "3 - Review"
+  color: "CCCCCC"
+  description: "Code has been added, and is available for review as a pull request"
+- name: "4 - Done"
+  color: "CCCCCC"
+  description: "Code has been added to the repository, and has been reviewed by a team member"
+- name: "5 - Released"
+  color: "CCCCCC"
+  description: "The issue has been resolved, and released to the public for consumption"
+- name: "ADD AUTO TESTS"
+  color: "8E44AD"
+  description: "Things that typically go to Test-Kitchen - once completed, IN REGRESSION TEST SUITE label added"
+- name: "ADD MANUAL TESTS"
+  color: "8E44AD"
+  description: "Manual tests has not been added for issue - once completed, IN MANUAL TEST SUITE label added"
+- name: "API"
+  color: "207DE5"
+  description: "Issues that handles API changes, not necessarily visible to end users"
+- name: "Bad Pull Request"
+  color: "EB6420"
+  description: "The pull request opened was not accepted, didn't follow the procedures, or was missing information"
+- name: "Blocked - External"
+  color: "5319E7"
+  description: "The issue or pull request is not fixable without a change in a upstream/external library"
+- name: "BreakingChange"
+  color: "5319E7"
+  description: "The issue will introduce backwards incompatible changes"
+  aliases: ["Breaking Change"]

```

Comment:
Question on this... what will the actual label be on the repository once the sync runs?  Will it be `Breaking Change` or `BreakingChange`?

Reply:
Once it has synced the label will be `BreakingChange`.

If a label of `Breaking Change` is found it will be renamed/updated to be `BreakingChange`